src/software/thunderscope/thunderscope_config.py

```diff
@@ -218,6 +229,14 @@ def configure_base_diagnostics(
             name="Logs",
             widget=setup_log_widget(**{"proto_unix_io": diagnostics_proto_unix_io}),
         ),
+        TScopeWidget(
+            name="Error Log",
+            widget=setup_robot_error_log_view_widget(
+                **{"proto_unix_io": current_proto_unix_io}
+            ),
+            position="below",
+            anchor="Logs",
+        ),
```

Comment:
this gets launched with thunderscope_main (no args for communication with robots). Would this make sense? @sauravbanna

Reply:
I think long term this makes sense. It would be nice to have key error messages show up (e.g. LOG(FATAL) in AI or new protobuf message for specifying unexpected scenarios in AI...).